packages/thumbnail-grid-block/src/components/Thumbnail/ThumbnailRichTextEditors.tsx

```diff
@@ -0,0 +1,53 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { DesignTokens, Thumbnail } from '../../types';
+import { RichTextEditor, convertToRteValue } from '@frontify/guideline-blocks-shared';
+import { titlePlugins } from '../../helper/plugins';
+import { Asset } from '@frontify/app-bridge';
+import { TextStyles } from '@frontify/fondue';
+
+type ThumbnailRichTextEditorsProps = {
+    designTokens: DesignTokens;
+    isEditing: boolean;
+    updateItemWith: (type: keyof Thumbnail, value: string | Asset[], id?: string) => void;
+    id?: string;
+    title?: string;
+    description?: string;
+};
+
+const DEFAULT_CONTENT_TITLE = convertToRteValue(TextStyles.ELEMENT_HEADING3);
+const DEFAULT_CONTENT_DESCRIPTION = convertToRteValue(TextStyles.ELEMENT_PARAGRAPH);
+
+export const ThumbnailRichTextEditors = ({
+    designTokens,
+    isEditing,
+    updateItemWith,
+    id,
+    title,
+    description,
+}: ThumbnailRichTextEditorsProps) => {
+    // TODO: memorize RTE for drag n drop: solved with the shared drag n drop
+    return (
+        <div style={{ width: '100%', wordBreak: 'break-word' }} data-test-id="thumbnail-rte">
+            <RichTextEditor
+                // id={`${id}-title`} // comment in after this one is merged: https://github.com/Frontify/fondue/pull/1350
+                designTokens={designTokens}
+                isEditing={isEditing}
+                onBlur={(value) => updateItemWith('title', value, id)}
+                value={title ?? DEFAULT_CONTENT_TITLE}
+                placeholder="Add a Title"
+                plugins={titlePlugins}
+                updateValueOnChange
+            />
+            <RichTextEditor
+                // id={`${id}-description`}
+                designTokens={designTokens}
+                isEditing={isEditing}
+                value={description ?? DEFAULT_CONTENT_DESCRIPTION}
```

Comment:
```suggestion
                value={description ?? convertToRteValue(TextStyles.ELEMENT_PARAGRAPH)}
```

And use them directly

Reply:
actually don't needed, as its paragraph. Thanks for pointing it out :D